zoo/checklists/steps.py

```diff
@@ -0,0 +1,56 @@
+from typing import Optional
+
+import attr
+from django.conf import settings
+import yaml
+
+
+@attr.s(frozen=True, slots=True)
+class Step:
+    tag: str = attr.ib()
+    category_name: str = attr.ib(cmp=False, repr=False)
+    id: str = attr.ib()
+    title: str = attr.ib(cmp=False, repr=False)
+    description: str = attr.ib(cmp=False, repr=False, default=None)
+    help_url: str = attr.ib(cmp=False, repr=False, default=None)
+    is_checked: Optional[bool] = attr.ib(default=None)
```

Comment:
What does None mean in this case? I'm having a hard time imagining an item that's neither complete nor incomplete. Maybe sometimes we have no way to tell? Could you add a comment explaining, if it's worth an explanation?